src/Tizen.NUI/src/public/Animation/KeyFrames.cs

```diff
@@ -103,6 +103,45 @@ public void Add(float progress, PropertyValue value, AlphaFunction alpha)
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Get the number of added key frame.
+        /// </summary>
+        /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public uint GetKeyFrameCount()
+        {
+            uint ret = Interop.KeyFrames.GetKeyFrameCount(SwigCPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return ret;
+        }
+
+        /// <summary>
+        /// Get the progress and value from specific key frame.
+        /// </summary>
+        /// <param name="index">The index of keyframe.</param>
+        /// <param name="progress">A progress value between 0.0 and 1.0.</param>
+        /// <param name="value">A value.</param>
+        /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void GetKeyFrame(uint index, out float progress, PropertyValue value)
+        {
+            Interop.KeyFrames.GetKeyFrame(SwigCPtr, index, out progress, PropertyValue.getCPtr(value));
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Set the value to specific key frame.
+        /// </summary>
```

Comment:
What happen if the index is not exist? It would be great if you give more description about this.

Reply:
Thank you. Nothing happened in that case. I wrote comment about it.